src/test/scala/chiseltest/experimental/tests/VcsCoverageTests.scala

```diff
@@ -0,0 +1,177 @@
+// SPDX-License-Identifier: Apache-2.0
+
+package chiseltest.experimental.tests
+
+import java.io.{ByteArrayOutputStream, File, PrintStream}
+
+import chisel3._
+import chiseltest._
+import chiseltest.experimental.TestOptionBuilder._
+import chiseltest.internal._
+import org.scalatest.flatspec.AnyFlatSpec
+import org.scalatest.matchers.should.Matchers
+
+class VcsCoverageTests extends AnyFlatSpec with ChiselScalatestTester with Matchers {
+  behavior of "Testers2"
+
+  it should "allow specifying line coverage for Vcs" in {
+    assume(firrtl.FileUtils.isVCSAvailable)
+
+    val outputStream = new ByteArrayOutputStream()
+    Console.withOut(new PrintStream(outputStream)) {
+      test(new Module {
```

Comment:
For all these tests, if you're not actually looking at the circuit behavior (and I don't see anything here that is doing that?), does it make sense to test on an empty module (perhaps there should even be an EmptyModule test utility class - more as a way of saying "this module intentionally left empty")? I'd prefer tests to be orthogonal - if the circuit stuff broke but this was fine, I'd prefer this not to fail.

And ditto for all the instances below.